src/java/za/co/mip/ablduck/javadoc/JavadocParser.java

```diff
@@ -0,0 +1,1797 @@
+// Generated from /home/roberte/Projects/pct/src/java/za/co/mip/ablduck/javadoc/JavadocParser.g4 by ANTLR 4.7
+package za.co.mip.ablduck;
+import org.antlr.v4.runtime.atn.*;
+import org.antlr.v4.runtime.dfa.DFA;
+import org.antlr.v4.runtime.*;
+import org.antlr.v4.runtime.misc.*;
+import org.antlr.v4.runtime.tree.*;
+import java.util.List;
+import java.util.Iterator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Iterator'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AUselessImportCheck)